pkg/helpers/helpers.go

```diff
@@ -164,3 +179,16 @@ func ListPools(node *corev1.Node, mcpLister v1.MachineConfigPoolLister) (*mcfgv1
 
 	return master, worker, custom, nil
 }
+
+// IsCoreOSNode checks whether the pretty name of a node matches any of the
```

Comment:
thought (non-blocking): I was originally going to suggest for this to be moved to `pkg/daemon/osrelease` before I noticed the `IsWindows()` function above it, which made me realize that this is a fine home for this function. However, seeing the `IsWindows()` function brings up a question about what we should do about OCL for Windows nodes since they can't be updated that way. At the very least, we may want to do a small spike to better understand that relationship.

(To be clear: I'm not asking you to make any changes; I'm just putting my thoughts here for posterity :smile:)